### PR TITLE
DS-811 Wrapper Element

### PR DESCRIPTION
**Pull request by MarcinMr**
## Jira

https://pegadigitalit.atlassian.net/browse/DS-811

## Summary

A new wrapper element is created.

## Details

A new wrapper is a basic element that contains a basic schema with `content` and `attributes`.

## How to test

Pull the branch. Check the wrapper docs and confirm it's a good equivalent of the old wrapper object.

## Release notes

The new `e-bolt-wrapper` will replace the old `o-bolt-wrapper` object.

### CODE REVIEW

**Review by colbytcook — APPROVED** [review]
Approved

**Review by cjwhitedev — APPROVED** [review]

File: docs-site/src/pages/pattern-lab/_patterns/20-elements/wrapper/05-wrapper.twig [hunk 1]
```diff
@@ -0,0 +1,110 @@
+{% set description %}
+    A wrapper has no semantic meaning. This element contains visual elements and content within itself.
```
Comment (danielamorse) [inline docs-site/src/pages/pattern-lab/_patterns/20-elements/wrapper/05-wrapper.twig:2]:
```suggestion
  A wrapper provides a max-width for any content passed into it. It has no semantic meaning on its own.
```

File: docs-site/src/pages/pattern-lab/_patterns/20-elements/wrapper/10-wrapper-use-case-custom-max-width.twig [hunk 2]
```diff
@@ -0,0 +1,47 @@
+{% set description %}
+  Whenever alternate width is needed use <code>attributes</code> prop and add <code>max-width</code> as an inline style.
```
Comment (danielamorse) [inline docs-site/src/pages/pattern-lab/_patterns/20-elements/wrapper/10-wrapper-use-case-custom-max-width.twig:2]:
```suggestion
  If a custom width is needed use <code>attributes</code> prop and add <code>max-width</code> as an inline style.
```

File: packages/elements/bolt-wrapper/README.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
+Allows to center content within a page
```
Comment (danielamorse) [inline packages/elements/bolt-wrapper/README.md:1]:
```suggestion
The Bolt Wrapper element provides a max-width for any content passed into it.
```

File: packages/elements/bolt-wrapper/src/wrapper.scss [hunk 4]
```diff
@@ -0,0 +1,31 @@
+/* ------------------------------------ *\
+   Wrapper
+\* ------------------------------------ */
+
+@import '@bolt/core-v3.x';
+
+/**
+ * Long-form comment.
```
Comment (danielamorse) [inline packages/elements/bolt-wrapper/src/wrapper.scss:8]: Please remove the placeholder comment.

File: packages/elements/bolt-wrapper/src/wrapper.scss [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+ * This spans multiple lines and is also constrained to no longer than 80
+ * characters in width.
+ *
+ * 1. Provide line-comments like this.
+ */
+
+$bolt-wrapper--max-width: bolt-breakpoint(xxlarge);
+
+.e-bolt-wrapper {
+  // Styles go here. /* [1] */
```
Comment (danielamorse) [inline packages/elements/bolt-wrapper/src/wrapper.scss:19]: Please remove the placeholder comment.

**Review by danielamorse — CHANGES_REQUESTED** [review]
@MarcinMr I left some copy suggestions. If this sounds good, please apply the changes and commit (you can [do this right in Github FYI](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/incorporating-feedback-in-your-pull-request#applying-suggested-changes)).

Otherwise, this looks great!

File: packages/elements/bolt-wrapper/src/wrapper.scss (hunk 4 above)
Comment (MarcinMr) [reply packages/elements/bolt-wrapper/src/wrapper.scss]: change request done

File: packages/elements/bolt-wrapper/src/wrapper.scss (hunk 5 above)
Comment (MarcinMr) [reply packages/elements/bolt-wrapper/src/wrapper.scss]: change request done

**Comment by MarcinMr** [comment]
@danielamorse approved your suggestions

**Review by danielamorse — APPROVED** [review]

**Pull request merged**